sql/testdata/select

```diff
@@ -120,3 +120,27 @@ SELECT *, "from", kw."from" FROM kw
 ----
 from from from
 1    1    1
+
+# SELECT from index.
+
+statement ok
+CREATE TABLE xyz (
+  x INT PRIMARY KEY,
+  y INT,
+  z INT,
+  CONSTRAINT foo INDEX (z, y)
+)
+
+statement ok
+INSERT INTO xyz VALUES (1, 2, 3), (4, 5, 6);
```

Comment:
nit: can you swap these tuples? it would call attention to the fact that the result is sorted by the index rather than insertion order